### DISCUSSION TITLE
Captions for thumbnails

### DISCUSSION THREAD

**Discussion by Ada-Si** [sachinchoolur/lightGallery#1163] [Q&A]
I've got implemented an inline gallery with thumbnails using dynamic mode. New request has come from our client to have captions on thumbnails. Unfortunately I haven't been able to find any way to accomplish that going through the official docs.
Is there any lightGallery way to do that?

Thank you

**Comment by dglangley** [comment]
I'm in need of this same feature, and I haven't been able to figure out if this is already supported using any current method.
